src/container/builder_impl.rs

```diff
@@ -21,6 +21,24 @@ use crate::{
 
 use super::{Container, ContainerStatus};
 
+struct ContainerInitArgs {
```

Comment:
Could you have the code placed a little closer to container_init() to make the relationship easier to understand? Or cutting it out into a file would be good too.

Reply:
I moved it closer to container_init().